### PR TITLE
List spots on the host page

### PR DESCRIPTION
**Pull request by nickcharles**
# What

This PR now has the host page doing something somewhat useful. It will display in a grid the user's current listings. This correctly hits the database and populates a grid view, however it looks terrible and we will need to make it a little prettier in the morning.

@kenzshelley @matthewgrossman

### CODE REVIEW

File: app/src/main/java/com/mmm/parq/adapters/SpotAdapter.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                textView = (TextView) convertView;
+            }
+
+            try {
+                JSONObject spotObj = mSpotsArray.getJSONObject(position);
+                JSONObject attrs = spotObj.getJSONObject("attributes");
+                String title = attrs.getString("title");
+                textView.setText(title);
+            } catch (JSONException e) {
+                Log.d("HOST", "Error parsing spots array");
```
Comment (kenzshelley) [inline app/src/main/java/com/mmm/parq/adapters/SpotAdapter.java:60]: Probs make this Log.e

**Comment by kenzshelley** [comment]
This looks good! I would be fine with merging this now, and then sending out a follow-up PR making the grid items fancier if there's time

**Comment by nickcharles** [comment]
Just pushed a version that has a _slightly_ fancier interface, check it out super quick and we can merge.

**Comment by kenzshelley** [comment]
lgtm

File: app/src/main/java/com/mmm/parq/activities/HostActivity.java [hunk 2]
```diff
@@ -23,6 +37,24 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        Firebase firebaseRef = new Firebase(getString(R.string.firebase_endpoint));
+        RequestQueue queue = Volley.newRequestQueue(this);
```
Comment (matthewgrossman) [inline app/src/main/java/com/mmm/parq/activities/HostActivity.java:45]: We have a singleton RequestQueue that kenz just told me about 

https://github.com/parq-app/parq-android/pull/26/files#diff-95d3da4048a6c24605814f59946c3f83R81

**Comment by matthewgrossman** [comment]
lgtm

File: app/src/main/java/com/mmm/parq/activities/HostActivity.java (hunk 2 above)
Comment (nickcharles) [inline app/src/main/java/com/mmm/parq/activities/HostActivity.java:45]: Will address later. #31

**Pull request merged**